src/Components/AdvancedSearch/Internal/QueryBuilder/QueryBuilder.tsx

```diff
@@ -146,52 +149,59 @@ const QueryBuilder: React.FC<IQueryBuilderProps> = (props) => {
 
     return (
         <div className={classNames.root}>
```

Comment:
hsouldn't we have a stack here to separate the query section from the buttons? THe buttons should ideally have space between them and it'd be great if that came from a stack vs styling on each button individually

Reply:
Stack would make the buttons flex to 100% width, unless I wrap them in a div and style that. I feel this is a simpler solution.

Reply:
I'm referring to this. 
![image](https://user-images.githubusercontent.com/57726991/189001026-bb871810-4c09-4b11-a0e7-f87e810761c6.png)
Which results in it looking like this.
![image](https://user-images.githubusercontent.com/57726991/189001041-7c0c46b5-6d2d-4259-ab1a-6c91169760ae.png)